ChatQnA/chatqna.py

```diff
@@ -66,9 +66,9 @@ def align_inputs(self, inputs, cur_node, runtime_graph, llm_parameters_dict, **k
         next_inputs["max_tokens"] = llm_parameters_dict["max_tokens"]
         next_inputs["top_p"] = llm_parameters_dict["top_p"]
         next_inputs["stream"] = inputs["streaming"]
-        next_inputs["frequency_penalty"] = inputs["frequency_penalty"]
-        next_inputs["presence_penalty"] = inputs["presence_penalty"]
-        next_inputs["repetition_penalty"] = inputs["repetition_penalty"]
+        next_inputs["frequency_penalty"] = inputs["repetition_penalty"]
+        # next_inputs["presence_penalty"] = inputs["presence_penalty"]
+        # next_inputs["repetition_penalty"] = inputs["repetition_penalty"]
```

Comment:
repetition_penalty， presence_penalty， repetition_penalty are three different parameters, need to pass them all for later usage, which are fixed in PR https://github.com/opea-project/GenAIComps/pull/608

Reply:
these parameters setting caused TGI accuracy issue, we need to check  why these parameters affect the accuracy.

Reply:
If the TGI accuracy change, please check and pass the correct value. For example, if you need "frequency_penalty=1.0", then you can set "frequency_penalty=1.0" directly, instead of setting "repetition_penalty=1.0" and pass it to “frequency_penalty”。

Reply:
As both Huggingface and Openai API are used in GenAIComps, the default value of repetition_penalty/presence_penalty/repetition_penalty can't guarantee optimal performance in all cases, please take care of the input parameters value in each usecase according to their underlying API implementation.

Reply:
TGI comps use different API (both Huggingface & OpenAI) for different input types https://github.com/opea-project/GenAIComps/blob/main/comps/llms/text-generation/tgi/llm.py. Maybe you need to turn on LOGFLAG=True, or print the parameters before transferring to LLM to check what's problem.

Reply:
@XinyaoWa thanks. Yes you are right, and we need to find the right three penalty parameters or raise a issue to TGI gaudi later if it doesn't work. For now we can just keep this PR as default. @lkk12014402 @lvliang-intel can we have a discussion about this issue?